### ISSUE TITLE
1 minute setting not working right

### ISSUE THREAD

**Issue by sydeburnz** [nickfox/GpsTracker#47]
looking at my database, it seems it is updating every 5 minutes even while set to update every 1 minute.

**Comment by sydeburnz** [comment]
hello?  anyone out there?

**Comment by brentfraser** [comment]
yep.  What is your configuration?  Which Phone (and O/S version)?  Which database engine and version?

**Comment by sydeburnz** [comment]
android 6.0.1 pushing to mysql  5.5.49-cll

**Comment by brentfraser** [comment]
we may need to update the Android app in the Play Store (and possibly the server components too).

**Comment by sydeburnz** [comment]
as in they are updated but you have to upload them?  or make changes to your end?  just curious as to timing since it is storm chasing season now :)

**Comment by brentfraser** [comment]
I think the problem you are having can be solved with a re-compile and an upload.  But we're also in the process of making changes to handle time zones so we may want to wait a couple of days until those changes are done then recompile and upload.  Hopefully I'll know more tomorrow.

**Comment by sydeburnz** [comment]
sweet.  im having fun playing with it.  i wrote a php page to extract data from the db and create a file that my radar program can ping to get coordinates and put us on radar map we use.  my buddy was out tonight and it was up to 10 minutes off though, so i thought to check back in on my original post.  will keep my eye out.

**Comment by brentfraser** [comment]
Interesting.  We're doing the time zone enhancements for a storm chaser site.  I'd be interested in what you are planning.

**Comment by sydeburnz** [comment]
we are not doing anything for public consumption.  just for our team.  if i use the spotter.org overlay on my radar, it is too cluttered when all the chasers are out descending on a storm.  this way, i can just show the 2 or 3 of us that may be out chasing.  since i can also publish the radar images to my server, we also have the ability to view a short loop with our locations as well.

**Comment by extremeweather1** [comment]
sydeburnz, we are doing a similar project http://xtremeweather.org love to hear from you and combine our efforts

**Comment by extremeweather1** [comment]
sydeburnz, we are doing a similar project http://xtremeweather.org and & http://xtremeweather.org/gpstracker love to hear from you and combine our efforts. We are still in development but will be combining the two in the next few days. Thanks to brent f. we have solved several of the issues when tracking chasers in next to real "Time" environment as well as known when a chaser are transmitting and not.

**Comment by sydeburnz** [comment]
just hit me up at jason@kswxlink.com

**Comment by radiantDavid** [comment]
Has any progress been made on this? I am also experiencing a 5 minute delay when checking my DB logs,

**Comment by nickfox** [comment]
The only way for us to test this is for you to send the data to the websmithing server. 

https://www.websmithing.com/gpstracker/updatelocation.php

Try sending data there for an hour or so and then let us know what your username is on the phone.

**Comment by nickfox** [comment]
@radiantDavid Did you see my what I said about websmithing?
